joomla-projects/privacy-framework
# Use popovers

Pull Request for Issue # .

### Summary of Changes

This replaces tooltips with popovers in terms and privacy fields.

### Testing Instructions

Go to registration page and check that popovers are displayed when hovering over terms and privacy field labels.

### Documentation Changes Required

No.
